docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -451,6 +469,22 @@ The following table lists the measurement data that the HRM sensor provides:
 | Timestamp                   | `unsigned long long` | -                                   | Microseconds |
 | values[0]: Beats per minute | `int`                | Min. value = 0<br> Max. value = 240 | -            |
 
+<a name="hrm_batch"></a>
+## Heart Rate Monitor Batch Sensor
```

Comment:
What is the is difference between
 Heart Rate Monitor Batch Sensor and Heart Rate Monitor Sensor?

The description is the same.
Can you please verify?

Reply:
Same as the above conversation. 

This PR was previously merged, but It was temporarily reverted by request of MCD.  
And with Tizen Studio 4.1, this guide should be released again.

Reply:
So, can we add to the description?